deploy/deploy_neo4j_server.yml

```diff
@@ -0,0 +1,24 @@
+# deploy_neo4j_server
+---
+- hosts: neo4j_server
+  become: yes 
+  roles:
+    - {role: puppet_disable, when: "disable_puppet"}
+    - {role: iptables, when: "iptables_config"}
+    - {role: common, when: "install_glusterfs"}
+    - {role: epel, when: "use_epel and not 'production' in group_names"}
+    - jdk
+    - neo4j 
+
+- hosts: neo4j_server 
```

Comment:
@olivermel These ports don't need to be open here, we only need the several ports for Neo4J itself, this range of ports was for applications running on app_server instances.